### ISSUE TITLE
The daily and weekly do not load

### ISSUE THREAD

**Issue by iLui** [Cvolton/GMDprivateServer#701]
I don't know what happens but when I want to put a daily and a weekly, they don't load.

**Comment by fancythedeveloper** [comment]
Did you not do !daily or !weekly on a level?

**Comment by iLui** [comment]
Yes, I do

**Comment by DarkGD123** [comment]
Wait for 1 day for a daily level then 7 days for weekly

**Comment by iLui** [comment]
I put a weekly and a week has passed and it does not start, neither does the newspaper, I put the newspaper two days ago and it does not load

**Comment by DarKSoulZ964** [comment]
Make yourself mod and set levels as daily, I think that's how, but I can't tell cause of bug in 7m.pl

**Comment by DarKSoulZ964** [comment]
WAit not mod make yourself dev

**Comment by fancythedeveloper** [comment]
Bro this is from January, lmao.

**Comment by DarKSoulZ964** [comment]
Make yourself dev in order to do it, here is tutorial: https://www.youtube.com/watch?v=Vnz7ZYSlXtI

**Comment by SoupBubble** [comment]
yeah, but it with all of that mentioned, daily still doesn't work
